repl.ts

```diff
@@ -34,8 +72,15 @@ export class BasicREPL {
     this.functions = "";
   }
   async run(source : string) : Promise<Value> {
+
+    // check if built-in classes are parsed
+    var sourceCode = source;
+    if (!this.currentEnv.classes.has("Range") || !this.currentEnv.classes.has("generator")) {
+      sourceCode = builtinClasses + source;
+    }
+
     const config : Config = {importObject: this.importObject, env: this.currentEnv, typeEnv: this.currentTypeEnv, functions: this.functions};
-    const [result, newEnv, newTypeEnv, newFunctions, instance] = await run(source, config);
+    const [result, newEnv, newTypeEnv, newFunctions, instance] = await run(sourceCode, config);
```

Comment:
This is okay for now, but we would not want to be appending to the source code before the user runs their code. If there are errors here, what would we report to the user?

Reply:
What would be an optimal way to add built-in function/classes to the program? In the future, should we append built-in code at the WASM level (e.g. inside `wasmSource` from `runner.ts`)?